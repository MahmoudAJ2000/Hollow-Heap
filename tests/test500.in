500 1500 100 156
78 135 929
94 335 49
119 385 835
387 104 191
465 375 523
78 345 977
181 99 788
304 206 213
156 281 240
153 226 346
87 421 294
349 314 628
334 447 873
95 57 783
234 489 380
253 234 400
431 302 20
294 374 522
78 224 743
444 7 860
215 448 618
275 212 424
478 266 67
76 19 981
32 454 226
61 19 106
320 13 206
356 378 398
67 280 347
102 319 2
439 62 868
284 149 839
257 79 986
485 486 366
211 212 567
50 155 653
129 16 499
345 474 992
204 298 880
211 303 39
436 166 520
491 103 460
475 433 204
99 87 59
284 399 158
145 233 615
119 349 415
386 131 32
8 176 773
202 56 348
70 178 253
431 247 850
270 162 834
278 66 811
492 141 680
238 353 847
15 6 544
310 475 959
288 186 153
270 406 958
171 290 656
483 36 814
114 497 704
74 272 810
1 332 277
4 89 570
273 96 4
311 439 428
263 128 530
352 218 334
13 348 364
130 108 856
1 94 15
365 369 621
231 315 700
254 411 272
132 385 608
437 445 501
205 390 49
486 360 910
80 84 664
490 33 698
49 146 335
443 181 790
88 300 788
352 358 693
333 453 383
380 456 205
334 308 628
322 276 192
233 323 23
491 135 622
268 441 258
491 190 507
330 217 257
418 121 218
116 84 530
288 469 382
56 153 950
334 154 431
444 186 50
420 450 502
411 275 45
91 173 224
16 13 989
448 180 744
207 259 128
293 449 481
1 465 619
158 304 956
308 54 43
280 371 610
364 297 603
50 342 47
323 221 636
301 182 622
188 312 313
100 105 843
73 486 919
361 74 410
218 36 659
458 262 733
41 108 125
38 473 125
76 427 657
54 152 887
415 106 454
204 313 667
286 453 897
218 338 587
60 215 681
161 210 367
95 152 330
349 107 969
164 258 697
107 308 58
203 103 974
293 456 94
363 58 283
339 494 960
324 81 270
487 391 673
406 349 644
37 366 382
268 451 686
264 328 950
57 6 172
38 311 574
161 314 377
333 168 731
476 120 287
356 129 339
255 231 188
92 279 884
157 245 831
440 340 927
375 279 241
458 119 856
458 90 955
25 374 784
430 246 634
284 455 410
118 111 744
244 6 161
275 292 860
309 361 60
184 373 103
351 498 92
261 57 467
221 52 918
103 353 471
469 231 656
134 155 707
479 107 817
115 461 78
235 4 832
333 0 283
182 125 621
141 429 908
378 470 182
487 162 451
15 102 364
23 10 347
255 68 130
291 496 269
329 361 340
235 91 96
476 77 409
277 71 849
235 356 325
11 58 922
461 430 587
398 355 292
472 68 401
114 364 739
119 112 13
217 125 725
146 363 393
445 328 464
280 370 202
331 317 326
347 217 30
164 450 530
417 319 440
11 257 580
0 267 86
351 160 262
370 12 881
129 436 447
379 492 940
126 351 190
280 1 227
344 400 1
353 482 21
384 470 636
134 180 239
284 113 109
498 182 491
277 40 760
51 123 88
438 235 841
2 205 858
447 146 616
92 348 308
76 9 568
101 27 660
172 95 302
333 299 107
376 445 828
80 411 168
496 250 204
402 373 94
481 219 895
498 125 664
241 363 489
96 18 100
105 492 137
167 140 67
96 299 390
28 465 863
276 327 274
282 431 213
483 199 805
46 32 76
26 449 640
16 237 248
409 132 586
335 333 920
120 26 33
369 45 148
349 222 789
48 142 218
250 14 23
153 75 211
164 139 335
94 250 462
108 100 673
97 282 498
98 234 902
320 251 439
144 77 531
442 425 823
106 499 438
256 105 652
12 250 288
143 441 318
401 157 217
312 58 906
335 423 266
79 108 782
179 2 248
85 8 108
209 136 482
354 125 518
62 214 783
169 312 456
132 121 386
209 134 277
391 325 518
137 482 558
440 150 407
221 69 784
104 283 551
317 56 649
158 256 63
121 211 125
20 186 518
195 397 862
192 321 354
298 404 284
335 355 774
177 370 131
92 67 245
384 142 516
248 5 707
377 379 168
370 87 200
449 402 880
315 31 169
180 156 311
466 349 35
84 226 909
315 249 958
7 135 613
478 380 426
202 212 523
74 70 477
320 392 615
271 99 948
175 148 690
150 411 489
247 45 491
322 331 144
398 383 743
147 172 626
412 304 189
235 304 609
385 105 30
213 94 531
473 476 794
464 122 685
224 183 279
365 217 123
235 73 932
430 184 661
474 99 943
107 153 328
217 182 26
382 376 382
299 165 382
282 387 289
402 291 827
232 380 723
367 165 857
445 428 975
439 243 323
59 355 348
308 161 157
253 289 153
64 341 428
3 217 279
146 256 221
162 135 601
438 35 512
467 465 797
444 296 723
458 101 65
8 392 549
92 424 920
105 199 574
198 375 976
357 425 83
234 130 967
304 485 74
449 229 252
475 359 543
340 314 506
490 371 247
411 268 769
182 154 12
332 447 460
196 393 907
416 9 995
315 362 618
201 369 820
84 27 53
246 393 312
17 44 611
64 453 209
268 379 820
47 64 997
272 38 309
168 232 496
303 188 160
84 294 480
257 322 109
95 224 483
259 116 756
384 349 646
253 4 184
294 275 263
199 345 416
269 101 82
24 58 790
406 81 40
493 433 153
452 104 928
10 297 369
442 79 136
277 363 818
258 418 504
367 88 408
467 427 408
242 448 716
274 323 237
479 211 943
309 64 454
459 363 175
449 394 546
106 199 815
0 327 361
147 467 495
451 383 955
453 354 647
210 306 95
145 268 178
490 488 16
76 3 252
174 340 862
59 281 981
28 409 264
100 317 666
312 56 403
436 249 249
356 19 846
145 99 426
119 238 973
205 33 275
50 9 309
196 136 355
343 251 136
463 439 840
490 286 753
184 209 220
26 458 680
448 195 342
40 366 221
79 458 26
377 443 85
330 22 822
127 179 899
367 391 816
478 256 70
267 448 761
99 305 308
391 22 443
172 343 251
241 329 537
260 269 655
372 469 268
204 321 945
46 344 471
145 231 72
254 79 570
95 59 799
145 439 104
65 42 492
191 226 693
16 363 546
352 175 354
10 36 231
35 478 50
431 47 529
181 101 258
486 483 145
431 245 372
448 400 716
339 437 257
55 89 482
110 348 945
442 33 701
234 403 756
345 221 685
206 63 0
482 210 850
277 472 398
336 379 529
414 303 75
5 394 132
168 31 348
257 112 249
436 188 598
255 58 310
456 280 235
121 484 291
414 204 157
113 30 873
360 338 359
441 71 355
168 76 946
19 219 109
32 147 325
10 195 335
373 173 880
462 482 495
105 261 70
15 318 120
350 228 628
102 355 298
254 233 756
258 436 790
19 215 726
7 119 118
264 5 90
125 198 851
389 201 383
70 301 202
105 168 876
479 387 440
106 77 791
74 5 446
342 259 690
364 330 511
290 56 432
470 424 556
349 38 149
457 487 419
211 487 857
136 98 118
3 342 262
122 62 623
370 314 206
14 53 296
82 13 574
120 196 801
435 243 891
166 444 302
59 497 811
223 70 296
60 370 343
214 56 778
333 226 327
22 92 228
180 410 54
14 490 470
258 314 763
494 184 204
225 227 807
200 88 532
319 494 569
490 112 373
71 461 232
420 75 863
70 226 476
388 192 866
93 231 746
16 296 516
276 21 580
131 319 671
168 201 869
461 97 541
102 130 173
316 194 656
274 394 370
333 261 242
97 10 341
477 314 893
349 466 350
241 443 263
30 84 82
73 415 488
493 121 541
292 170 820
368 336 110
307 287 683
442 226 22
169 195 878
195 491 71
156 442 420
51 239 681
0 459 783
309 436 259
494 286 809
32 421 693
22 19 206
64 467 185
86 459 849
231 143 128
464 460 844
71 223 382
276 349 789
454 242 599
60 205 412
474 365 623
112 336 774
1 334 615
49 387 381
135 418 514
81 265 913
298 217 316
143 10 390
256 68 265
400 309 979
495 460 353
317 499 814
353 128 992
160 290 179
436 385 858
200 401 458
274 358 567
17 340 643
79 348 914
153 73 698
8 87 319
133 253 881
115 244 654
392 33 614
281 447 303
405 46 480
22 203 426
90 158 616
179 162 324
193 302 425
317 24 635
491 7 827
413 188 846
345 405 948
57 465 36
374 262 419
216 60 291
192 159 178
298 94 302
484 239 480
234 53 840
151 484 877
232 214 301
448 487 94
444 81 261
476 462 308
146 405 12
104 291 979
485 480 285
432 118 338
30 169 621
435 113 527
413 397 964
195 225 465
318 322 836
50 292 489
467 473 79
35 9 501
475 352 994
303 179 717
44 445 75
102 111 75
466 294 883
208 370 210
208 177 818
118 71 417
231 115 181
478 292 491
105 218 86
258 439 446
457 218 673
208 277 689
206 388 301
75 432 609
303 325 772
166 408 49
241 106 302
366 462 840
76 171 828
159 196 741
352 155 962
405 328 582
16 263 610
170 428 54
5 316 73
104 331 332
428 35 324
413 309 493
193 411 353
238 441 646
17 124 611
381 145 233
301 68 728
370 462 639
495 324 437
3 205 241
155 41 880
485 96 11
71 62 354
405 16 746
300 47 867
3 196 876
457 103 17
483 66 359
178 194 821
268 85 292
423 152 796
302 400 280
496 409 672
122 172 393
194 193 676
436 89 410
435 386 324
490 25 837
21 248 370
203 298 974
487 1 353
425 194 895
116 37 504
138 375 238
259 309 449
406 233 868
403 417 932
258 325 826
495 38 915
396 369 116
237 321 81
413 482 742
236 343 191
189 1 22
42 467 773
95 217 97
1 145 129
55 37 295
257 335 924
327 142 938
29 273 389
227 224 184
258 197 169
43 388 750
81 471 722
366 433 942
28 332 420
499 360 623
141 350 170
195 354 417
373 96 437
354 465 268
168 378 8
30 333 288
398 462 75
426 339 148
205 51 429
311 74 384
481 111 910
415 315 821
27 399 409
176 320 679
165 235 700
166 358 469
7 212 453
49 470 957
449 432 51
132 394 492
126 169 749
8 421 174
166 41 166
169 155 457
287 159 796
191 408 609
322 39 780
159 410 495
88 268 42
271 303 318
44 297 412
414 173 978
41 394 49
310 377 263
74 409 155
292 446 366
100 103 63
235 278 427
227 111 317
157 110 518
385 133 138
403 373 360
419 78 55
437 76 931
175 237 449
127 488 555
239 43 243
438 8 110
259 301 442
434 106 932
154 209 326
334 236 151
245 193 461
374 177 519
136 194 532
294 22 4
170 308 994
442 284 882
222 166 465
451 201 517
212 414 556
291 476 313
260 254 957
193 164 184
224 378 637
238 257 810
51 281 40
218 110 901
383 36 981
8 353 682
340 312 16
200 30 653
258 109 756
280 9 835
11 107 501
218 95 335
485 425 326
40 351 73
195 72 791
330 478 242
117 347 257
170 116 359
430 488 209
232 366 904
67 418 599
80 125 41
344 0 388
305 374 485
142 412 834
435 58 380
368 441 980
75 384 439
194 251 190
213 26 584
431 144 55
89 79 435
227 53 105
377 334 255
350 95 96
199 482 633
305 45 108
470 409 626
326 438 533
447 307 831
439 33 698
209 438 183
423 90 306
242 351 368
379 194 219
47 224 464
322 291 142
396 24 46
358 94 688
101 398 113
377 454 938
163 191 386
351 314 368
106 462 152
94 338 646
49 239 132
129 165 803
58 326 298
132 114 621
388 274 658
460 207 577
142 133 937
105 429 864
298 483 372
246 137 80
310 165 460
445 294 357
222 232 496
297 146 541
472 159 363
165 489 544
472 392 160
426 244 118
497 203 638
157 300 43
136 420 239
475 301 268
372 174 806
445 333 919
94 19 513
341 3 450
243 56 789
228 389 633
216 242 661
90 43 519
117 151 699
201 82 835
437 466 499
300 286 419
392 353 543
151 173 490
410 245 398
121 472 89
255 354 146
449 194 977
451 292 390
230 119 810
50 331 38
108 247 152
328 230 703
423 76 751
237 239 964
227 379 405
297 456 926
232 348 823
26 211 357
302 430 946
302 144 651
60 193 781
111 232 745
453 98 145
323 38 290
350 151 824
183 19 540
148 315 605
136 397 808
260 238 777
40 303 112
448 88 752
399 498 648
455 115 746
37 447 321
60 497 831
127 87 928
209 429 619
397 181 64
283 170 133
1 112 981
339 298 656
181 117 328
74 350 381
394 220 944
358 312 334
287 220 222
244 256 480
425 95 69
276 287 22
494 200 436
115 145 23
285 282 617
322 471 387
193 136 587
146 336 313
271 106 471
328 109 405
32 474 540
150 350 327
377 302 184
280 201 191
338 480 938
247 341 375
156 111 982
318 416 913
48 101 295
446 478 86
45 171 728
182 316 695
326 86 821
312 131 843
405 486 468
396 359 580
286 315 755
347 224 900
269 307 382
30 69 486
296 63 929
77 255 690
325 440 904
13 307 824
207 205 205
279 388 709
484 491 846
177 379 626
122 147 812
336 63 105
84 228 431
115 46 984
260 327 814
422 26 895
261 484 500
164 243 90
275 320 427
259 105 799
375 28 895
152 379 219
162 248 653
169 28 428
478 269 919
427 4 584
475 418 190
224 35 21
52 142 515
76 376 219
103 69 471
199 200 253
492 451 341
484 245 875
338 438 562
309 441 770
75 32 987
177 104 907
240 52 846
203 369 308
273 81 631
88 141 28
250 252 189
80 454 176
379 223 132
457 270 916
218 288 202
241 73 624
135 460 993
257 225 590
53 333 774
488 79 302
21 1 233
64 57 225
357 1 922
5 445 919
316 180 372
483 207 630
401 111 271
246 143 637
316 214 611
423 297 633
449 29 975
234 417 24
254 35 707
95 424 103
406 285 420
134 83 111
130 473 624
3 424 300
446 453 437
364 457 784
368 188 1
322 259 349
258 247 806
230 87 946
36 255 27
354 471 381
247 407 105
65 119 864
201 110 255
219 463 906
65 80 901
376 361 16
391 433 683
283 304 491
246 199 263
448 152 501
39 241 415
333 251 711
195 63 56
265 267 776
26 232 125
289 0 796
58 273 948
102 99 63
230 153 742
359 337 707
153 480 671
188 31 566
376 93 148
471 193 855
212 258 111
231 367 968
419 152 99
37 48 988
49 101 32
271 242 385
240 187 787
92 431 422
146 239 717
92 272 964
320 162 694
214 77 988
28 76 759
121 422 529
428 379 574
9 293 119
484 402 126
57 421 391
463 369 100
336 74 959
280 29 723
166 70 260
129 222 661
140 442 875
452 484 215
396 489 215
331 72 428
412 89 30
13 268 771
19 182 602
17 400 930
105 398 90
7 311 207
487 443 264
124 393 217
301 232 912
422 236 852
81 101 193
244 257 239
103 426 218
358 265 73
7 134 135
452 27 643
460 83 782
363 244 812
398 314 623
91 437 530
94 290 492
118 491 267
464 201 551
306 107 67
243 158 786
9 190 403
126 335 556
402 189 384
366 482 617
121 227 370
351 447 875
403 106 464
292 490 482
459 395 886
219 413 947
64 20 646
63 171 405
499 81 573
303 321 610
319 155 591
45 25 10
317 398 877
421 304 977
403 222 752
308 64 475
379 40 822
111 395 316
226 414 808
104 96 980
61 455 217
184 488 529
132 37 691
477 24 901
464 340 629
469 482 486
162 50 308
366 437 82
152 274 930
298 214 45
17 291 101
304 446 567
171 164 863
201 343 851
297 288 224
75 54 265
69 491 50
104 323 656
82 116 506
234 109 598
405 452 427
432 291 106
305 72 18
395 409 248
66 273 185
170 138 560
125 245 141
248 392 760
492 249 559
340 440 440
23 359 377
439 26 797
202 100 493
432 140 239
86 319 236
166 478 895
414 300 49
137 43 354
98 160 16
124 52 484
120 33 622
163 40 943
181 58 320
83 269 448
161 160 811
477 187 89
348 231 273
69 40 713
468 71 240
355 390 805
183 110 832
389 76 761
178 214 549
481 150 5
203 399 451
220 178 426
140 86 567
84 192 251
267 390 592
302 489 268
34 473 157
183 111 684
34 336 731
329 153 656
137 144 279
458 197 247
154 462 118
215 395 891
439 492 122
223 27 421
449 62 800
395 200 117
356 233 822
180 327 497
376 386 397
392 62 396
381 403 447
405 24 514
207 256 427
80 54 234
324 443 812
246 158 294
62 309 605
323 69 266
482 465 550
95 29 933
297 56 49
354 33 364
270 64 131
388 497 243
301 41 622
264 30 700
355 438 915
299 117 832
5 141 363
159 116 79
77 90 382
256 58 833
194 249 911
20 286 334
123 81 895
47 406 821
481 488 308
292 213 128
153 138 517
35 402 547
276 127 174
205 141 290
412 463 271
492 70 602
117 73 692
115 281 333
390 174 719
303 432 566
413 326 865
380 375 485
198 381 585
369 361 98
78 239 600
188 60 59
89 141 681
455 127 758
130 310 566
58 337 388
306 185 473
196 331 250
167 462 720
495 264 618
2 457 796
72 83 246
68 463 351
473 35 973
226 200 681
399 490 703
28 391 179
226 246 382
102 229 157
134 468 142
133 182 838
136 354 858
110 275 253
318 463 63
161 79 872
329 41 646
61 390 276
225 142 171
263 297 251
172 85 752
83 271 965
459 116 878
234 388 928
118 96 810
148 280 417
466 242 442
253 464 774
169 405 9
22 381 109
205 178 839
340 200 92
277 452 795
45 189 760
352 10 902
335 414 40
339 128 137
423 397 170
169 472 594
204 182 325
299 272 725
335 26 406
482 98 12
292 148 118
176 455 862
474 293 857
420 77 420
57 317 951
216 154 964
327 265 105
211 484 131
293 278 813
412 16 278
97 204 444
482 197 386
81 443 758
59 259 475
318 13 496
331 211 185
13 52 328
91 81 796
439 7 111
235 26 976
460 174 418
270 11 127
455 247 942
49 306 452
93 88 703
142 34 467
83 50 82
470 29 283
26 113 208
226 122 23
264 209 842
155 76 871
9 415 777
441 393 89
79 433 639
30 240 235
429 215 70
231 61 343
196 444 924
140 267 681
319 481 309
116 83 395
211 483 168
46 137 860
498 322 981
246 31 856
222 493 303
247 405 41
448 456 673
317 430 73
213 388 813
386 498 380
168 396 707
464 10 591
330 5 115
298 178 493
243 164 971
318 496 377
99 247 148
210 191 848
0 308 809
244 237 223
103 48 364
120 311 457
8 194 419
221 395 406
286 135 301
135 168 315
442 436 52
161 205 410
347 263 915
13 263 720
386 43 594
201 425 325
365 199 837
181 390 406
375 493 223
15 109 814
63 429 221
226 216 562
383 410 758
95 45 138
322 136 703
148 128 669
477 150 82
285 71 94
293 110 818
470 433 795
460 253 425
36 333 11
86 172 329
89 495 525
430 311 926
95 43 381
383 378 787
172 350 715
432 98 107
491 203 720
48 238 155
408 205 272
380 217 1
302 271 128
94 275 485
278 304 186
12 360 142
384 448 892
458 464 918
180 370 795
115 401 648
154 405 379
133 8 978
105 198 229
366 336 384
35 235 383
425 461 572
117 307 579
288 283 368
172 74 184
63 305 741
321 350 179
295 261 946
189 446 229
446 304 151
485 102 676
301 248 308
248 9 338
144 218 440
317 211 429
130 138 771
255 149 154
167 400 496
289 364 69
192 106 950
351 446 254
441 464 365
387 288 903
267 369 411
17 37 37
345 105 983
264 122 759
117 231 852
389 73 453
437 261 682
25 90 733
368 350 807
488 294 614
32 489 643
137 37 337
145 493 574
277 264 459
382 272 463
351 409 474
497 302 788
318 383 570
2 445 759
279 338 164
69 348 356
259 98 423
349 94 356
181 192 919
219 328 728
260 337 242
261 135 369
376 373 958
331 446 832
225 356 939
307 397 39
17 327 779
30 437 171
410 236 824
177 275 113
286 289 178
68 25 367
242 488 0
427 164 289
216 269 877
258 186 861
385 76 907
328 407 714
18 37 818
169 411 673
180 49 757
207 233 621
179 193 746
117 296 178
220 77 760
212 218 85
310 151 904
8 295 629
144 217 953
491 256 459
306 283 124
84 169 538
276 484 877
61 124 158
374 113 339
347 205 296
77 79 342
217 18 619
454 11 75
490 416 407
277 191 796
199 258 796
172 285 47
359 105 260
483 93 925
62 412 473
273 480 119
31
340
424
481
58
195
351
280
450
236
390
346
445
64
323
478
337
330
442
494
241
106
267
157
357
465
375
89
108
170
439
7
95
467
111
379
143
392
245
313
381
167
59
414
274
217
370
145
49
477
120
39
485
97
368
325
224
130
83
273
452
219
288
263
281
380
416
19
360
163
374
115
386
138
216
174
286
430
402
127
403
152
6
301
186
279
199
461
348
28
336
47
72
209
29
344
136
265
483
354
