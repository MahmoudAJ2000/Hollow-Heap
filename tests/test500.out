Distance from 156 to 31 is 1636
Distance from 156 to 340 is 2129
Distance from 156 to 424 is 1263
Distance from 156 to 481 is 2811
Distance from 156 to 58 is 1797
Distance from 156 to 195 is 2186
Distance from 156 to 351 is 2033
Distance from 156 to 280 is 2548
Distance from 156 to 450 is 2624
Distance from 156 to 236 is 2811
Distance from 156 to 390 is 1362
No path from 156 to 346 was found
Distance from 156 to 445 is 2206
Distance from 156 to 64 is 2147
Distance from 156 to 323 is 2585
Distance from 156 to 478 is 1714
Distance from 156 to 337 is 2185
Distance from 156 to 330 is 2476
Distance from 156 to 442 is 420
Distance from 156 to 494 is 3071
Distance from 156 to 241 is 3898
Distance from 156 to 106 is 2354
Distance from 156 to 267 is 2310
Distance from 156 to 157 is 1798
No path from 156 to 357 was found
Distance from 156 to 465 is 1880
Distance from 156 to 375 is 2403
Distance from 156 to 89 is 882
Distance from 156 to 108 is 1283
Distance from 156 to 170 is 2149
Distance from 156 to 439 is 2051
Distance from 156 to 7 is 2154
Distance from 156 to 95 is 1160
Distance from 156 to 467 is 1772
Distance from 156 to 111 is 982
Distance from 156 to 379 is 1709
Distance from 156 to 143 is 1461
Distance from 156 to 392 is 2267
Distance from 156 to 245 is 2045
Distance from 156 to 313 is 2074
Distance from 156 to 381 is 2174
No path from 156 to 167 was found
Distance from 156 to 59 is 1959
Distance from 156 to 414 is 1250
Distance from 156 to 274 is 2420
Distance from 156 to 217 is 1257
Distance from 156 to 370 is 1472
Distance from 156 to 145 is 1947
Distance from 156 to 49 is 2690
No path from 156 to 477 was found
Distance from 156 to 120 is 2597
Distance from 156 to 39 is 3483
Distance from 156 to 485 is 1824
Distance from 156 to 97 is 1772
No path from 156 to 368 was found
Distance from 156 to 325 is 2097
Distance from 156 to 224 is 1643
Distance from 156 to 130 is 2673
Distance from 156 to 83 is 2009
Distance from 156 to 273 is 2328
Distance from 156 to 452 is 1598
Distance from 156 to 219 is 2301
Distance from 156 to 288 is 1648
Distance from 156 to 263 is 2449
Distance from 156 to 281 is 240
Distance from 156 to 380 is 2140
Distance from 156 to 416 is 2570
Distance from 156 to 19 is 2192
Distance from 156 to 360 is 2495
No path from 156 to 163 was found
Distance from 156 to 374 is 2092
Distance from 156 to 115 is 1924
Distance from 156 to 386 is 2853
Distance from 156 to 138 is 2709
Distance from 156 to 216 is 1004
Distance from 156 to 174 is 2081
Distance from 156 to 286 is 1718
Distance from 156 to 430 is 1818
Distance from 156 to 402 is 1939
Distance from 156 to 127 is 2470
Distance from 156 to 403 is 2621
Distance from 156 to 152 is 1490
Distance from 156 to 6 is 2069
Distance from 156 to 301 is 1454
Distance from 156 to 186 is 1344
Distance from 156 to 279 is 2644
Distance from 156 to 199 is 1087
Distance from 156 to 461 is 1231
Distance from 156 to 348 is 1470
Distance from 156 to 28 is 2824
Distance from 156 to 336 is 1472
Distance from 156 to 47 is 2166
Distance from 156 to 72 is 2336
Distance from 156 to 209 is 1621
Distance from 156 to 29 is 2093
Distance from 156 to 344 is 2122
Distance from 156 to 136 is 2103
Distance from 156 to 265 is 1534
Distance from 156 to 483 is 1784
Distance from 156 to 354 is 2255
